.github/workflows/gradle.yml

```diff
@@ -89,27 +50,17 @@ jobs:
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v4
 
-      - name: Set up JDK 11
-        uses: actions/setup-java@v4
-        with:
-          java-version: '11'
-          distribution: 'temurin'
-
-      - name: ${{ matrix.project }} generate code for ${{ matrix.tag }}
+      - name: Override properties
+        if: matrix.project == 'node' || matrix.project == 'java'
```

Comment:
Just spotted this: `node` and `java` OpenAPI generator versions override the default one, but this should be also the case for Ruby, Python and (soon) .NET as they are also upgraded.
Maybe we can invert the approach: update the default OpenAPI Generator version to v7.x.x, and override the libraries (php, go) that are still behind? We are upgrading those libraries too, so we should choose what's easiest and quickest now (it can also be a different PR)